Dockerfile

```diff
@@ -30,7 +30,6 @@ php-pear curl \
 php5-json \
 php5-redis \
 php5-memcache \
-memcached \
```

Comment:
Si, lo añadí pensando que era necesario, pero noté que en la implementación original no se utiliza